src/app/item-review/dtr-api-types.ts

```diff
@@ -0,0 +1,71 @@
+/** Information about the reviewer. Consistent between D1 and D2. */
+export interface DtrReviewer {
+  membershipType: number;
+  membershipId: string;
+  displayName: string;
+}
+
+/** A user's review returned from DTR. */
+export interface DimUserReview {
+  /**
+   * This is not returned from DTR, it's calculated on our end.
+   * Will be set on reviews associated with any other reviwer that the user reports.
+   */
+  isIgnored?: boolean;
+  /**
+   * Is this reviewer a featured reviewer?
+   * Broken in D2.
+   */
+  isHighlighted: boolean;
+  /**
+   * Was this review written by the DIM user that requested the reviews?
+   * Broken in D2.
+   */
+  isReviewer: boolean;
+  /**
+   * Pros.
+   * Shouldn't be present (yet).
+   */
+  pros: string;
+  /**
+   * Cons.
+   * shouldn't be present (yet).
+   */
+  cons: string;
```

Comment:
I feel like I've seen the same set of fields in a bunch of different types. Suggests there's still more that can be done to clean these up.